src/pages/partners/affiliate-ib/index.tsx

```diff
@@ -165,7 +165,7 @@ const AffiliateIb = () => {
         {
             title: content_data.payout_title,
             subtitle: content_data.payout_text,
-            image_name: 'daily_ib_commission',
+            image_name: is_eu ? 'daily_ib_commission_eu' : 'daily_ib_commission',
```

Comment:
can you check this , on eu  ON Reload , there is a possibility it will first show  daily_ib_commission 
and then after some time it will render daily_ib_commission_eu 
can you just double confirm ?

Reply:
```suggestion
            image_name: is_region_loading ?(is_eu ? 'daily_ib_commission_eu' : 'daily_ib_commission'):null,
```

Reply:
Checked on this it only show daily_ib_commission_eu on eu.